specification/applicationinsights/resource-manager/microsoft.insights/preview/2017-10-01/eaSubscriptionMigration_API.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
I don't find the private version of this file in the two private PRs you mention? Am I missing something?

Reply:
Ok found it